### DISCUSSION TITLE
Self deployment not counting private repos

### DISCUSSION THREAD

**Discussion by arribass** [anuraghazra/github-readme-stats#1024] [Q&A] [answered]
As stated in readme documentation if you deploy your own instance private contributions will be counted. I've followed the deploy tutorial step by step. Am i missing anything?

**Comment by rickstaa** [comment, marked as answer]
@arribass I hope you were able to solve your problem. If not, the most likely cause is that something went wrong trying to create a Personal Access Token and adding this token your vercel instance (see https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance for more information).

**Reply by rickstaa** [reply]
@arribass Did that answer your question?

**Reply by arribass** [reply]
It did thanks!

**Reply by rickstaa** [reply]
Great to hear! Can you maybe accept my answer so that this question shows up as answered? Sadly, if I accept it myself, it will prevent my [galaxy-brain](https://github.com/rickstaa?achievement=galaxy-brain&tab=achievements) bardge from showing up due to a GitHub bug. 😆
